src/Razor/src/rzls/Program.cs

```diff
@@ -49,16 +55,43 @@ public static async Task Main(string[] args)
                     await Console.Error.WriteLineAsync($"Invalid Razor trace '{traceArg}'. Defaulting to {trace}.").ConfigureAwait(true);
                 }
             }
+
+            if (args[i] == "--telemetryLevel" && i + 1 < args.Length)
+            {
+                telemetryLevel = args[++i];
+            }
+
+            if (args[i] == "--sessionId" && i + 1 < args.Length)
+            {
+                sessionId = args[++i];
+            }
+
+            if (args[i] == "--sharedDependencies" && i + 1 < args.Length)
```

Comment:
@ryzngard Roslyn has this parameter so I included it in Razor, but I'm not 100% sure of a practical use case - would you have any insight here?

Reply:
This is because Roslyn has multiple extensions it has to load, most notably debugger and telemetry packages. For Razor we can simplify and just take the path to the telemetry assembly